lib/openshift/network.rb

```diff
@@ -0,0 +1,35 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift environment Network class
+  class Network < ClusterResource
```

Comment:
Now I can't say for sure because I don't know well these concepts. But it doesn't sound like `Network` can be a parent class to both `NetworkOperator` and `NetworkConfig`. According to https://en.wikipedia.org/wiki/Behavioral_subtyping

If you want to share method between both classes, I'd suggest you to write a `module` in the `flakes` directory, then `include` that module into both classes. Please use separate files for all cluster resources, otherwise some of the automatic fucntionality in `lib/resources.rb` is broken.

Example for using `module` see `Helper` module inside  `lib/common.rb` which is included in many places.

In case I am mistaken about inheritance above, it's alright to have class and inheritance, still in separate files though.

Reply:
I think we really only care about the NetworkOperator because that is what is active.  I'll check with @anuragthehatter

Reply:
Yep, I agree. We can just care about Network Operator.